images/scripts/rhel.setup

```diff
@@ -32,6 +32,31 @@ SKIP_REPO_FLAG="/root/.skip_repos"
 #
 set -x
 
+# Perform additional settings if rel-eng compose is used
+if [[ "$IMAGE" =~ ^rhel(-[[:digit:]]){2}(-([[:digit:]]{8}-[[:digit:]]|latest))$ ]]; then
+    major=${IMAGE:5:1}
+    minor=${IMAGE:7:1}
+    compose_build=${IMAGE:9}
+    compose_build=${compose_build/-/.}
+    if [[ "$IMAGE" =~ ^rhel-8-.-.{8}-.?$ ]]; then
+        compose="RHEL-$major.$minor.0-$compose_build"
+    elif [[ "$IMAGE" =~ ^rhel-7-.-.{8}-.?$ ]]; then
+        compose="RHEL-$major.$minor-$compose_build"
+    else
+        compose="latest-RHEL-$major.$minor"
+    fi
```

Comment:
Please no, this is too much and brittle string manipulation hackery. Please instead pass the repo URLs (which your other script already computed) as environment variables, and change image-create's `run_setup_script` to add them to the `env` variable.

Reply:
I simplified the script and did what you suggested in this comment, hopefully this is (close enough to) how you'd expect the script (and modifications to the existing tooling) to look like.